openedx_learning/core/contents/migrations/0001_initial.py

```diff
@@ -1,103 +1,75 @@
-# Generated by Django 3.2.18 on 2023-05-11 02:07
+# Generated by Django 3.2.19 on 2023-06-15 14:43
 
 import django.core.validators
 from django.db import migrations, models
 import django.db.models.deletion
+import openedx_learning.lib.fields
 import openedx_learning.lib.validators
 
 
+def use_compressed_table_format(apps, schema_editor):
+    """
+    Use the COMPRESSED row format for TextContent if we're using MySQL.
+
+    This table will hold a lot of OLX, which compresses very well using MySQL's
+    built-in zlib compression. This is especially important because we're
+    keeping so much version history.
+    """
+    if schema_editor.connection.vendor == 'mysql':
+        table_name = apps.get_model("oel_contents", "TextContent")._meta.db_table
+        sql = f"ALTER TABLE {table_name} ROW_FORMAT=COMPRESSED;"
+        schema_editor.execute(sql)
```

Comment:
For any reviewers: this is the main new thing here, manually added.

Reply:
Cool! Good idea.